src/server/ndjsonWorker.js

```diff
@@ -41,8 +45,16 @@ ndjsonWorker.process(async job => {
       const data = JSON.parse(resourceStr);
       return updateResource(data.id, data, data.resourceType);
     });
+  let count = await getCurrentSuccessfulImportCount(clientId);
+  const results = await Promise.all(insertions);
 
-  await Promise.all(insertions);
+  results.map(result => {
+    if (result.created === true) {
+      count = count + 1;
+    }
+  });
```

Comment:
This block of code will not work in the case of importing resources that already exist.

You shouldn't have to iterate over the results, and you can just use `results.length` as the number of successful insertions.

This loop results in count being `0` every time if we repeat the same import call multiple times since it will be doing `update` instead of `create` (created is always false then)